### PR TITLE
guile-gcrypt: init at 0.2.1

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by MagicRB — APPROVED** [review]
This looks fine too (when it builds of course)

**Comment by ethancedwards8** [comment]
It builds fine on my machine. Not sure why ofborg fails

File: pkgs/development/guile-modules/guile-gcrypt/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    sha256 = "1qj1yw0kman984x584jjjxnjdhm0pwgp09iyn3b5rqajx7svpqcd";
+  };
+
+  postConfigure = ''
+    sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+    sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
+  '';
+
+  nativeBuildInputs = [
+    autoreconfHook pkgconfig texinfo
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-gcrypt/default.nix:26]:
```suggestion
    autoreconfHook pkg-config texinfo
```

File: pkgs/development/guile-modules/guile-gcrypt/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ];
+  buildInputs = [
+    guile
+  ];
+  propagatedBuildInputs = [
+    libgcrypt
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-gcrypt/default.nix:32]: Is that always required when using guile-gcrypt anywhere and would there need to be listed under buildInputs?

File: pkgs/development/guile-modules/guile-gcrypt/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+, autoreconfHook
+, pkgconfig
+, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  src = fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-gcrypt/default.nix:15]: Can we use fetchFromGitea instead?

File: pkgs/development/guile-modules/guile-gcrypt/default.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, libgcrypt
+, autoreconfHook
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-gcrypt/default.nix:7]:
```suggestion
, pkg-config
```

File: pkgs/development/guile-modules/guile-gcrypt/default.nix (hunk 3 above)
Comment (ethancedwards8) [reply pkgs/development/guile-modules/guile-gcrypt/default.nix:15]: I'm not sure. From looking at the interface I think it's Gogs instead. I know Gitea is a Gogs fork but would they be compatible?

Comment (ethancedwards8) [reply pkgs/development/guile-modules/guile-gcrypt/default.nix:15]: I'm not sure. From looking at the interface I think it's Gogs instead. I know Gitea is a Gogs fork but would they be compatible?

Comment (jonringer) [reply pkgs/development/guile-modules/guile-gcrypt/default.nix:15]: most likely

File: pkgs/development/guile-modules/guile-gcrypt/default.nix (hunk 2 above)
Comment (ethancedwards8) [reply pkgs/development/guile-modules/guile-gcrypt/default.nix:39]: I believe it is required.

**Comment by ethancedwards8** [comment]
Fixed, thanks.

File: pkgs/development/guile-modules/guile-gcrypt/default.nix [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  # src = fetchurl {
+  #   url = "https://notabug.org/cwebber/${pname}/archive/v${version}.tar.gz";
+  #   sha256 = "1qj1yw0kman984x584jjjxnjdhm0pwgp09iyn3b5rqajx7svpqcd";
+  # };
```
Comment (SuperSandro2000) [inline pkgs/development/guile-modules/guile-gcrypt/default.nix:18]:
```suggestion
```

Comment (ethancedwards8) [reply pkgs/development/guile-modules/guile-gcrypt/default.nix]: Oops.

**Pull request merged**